draft-ietf-oauth-jwk-thumbprint-uri.xml

```diff
@@ -102,10 +102,21 @@
 	</list>
       </t>
       <t>
-	The prefix is followed by a colon and a JWK Thumbprint value to form
-	a URI representing a JWK Thumbprint.
+	The prefix is followed by a hash algorithm identifier and a JWK Thumbprint value, 
+  separated by the colon to form a URI representing a JWK Thumbprint.
       </t>
 
+      <section anchor="HashAlgorithms" title="Hash Algorithms Identifier">
+  <t>
+  Hash algorithm identifier used in JWK Thumbprint URIs are strings 
+  registered in the IANA "Named Information Hash Algorithm Registry" <xref target="IANA.Hash.Algorithms"/>.
+  </t>
+
+  <section anchor="MTI" title="Mandatory to Implement Hash Algorithm">
+	<t>
+	  Mandatory to implement Hash Algorithm is SHA-256.
+	</t>
```

Comment:
I haven't tried compiling it yet, but I think you're missing </section> after the </t>.

Reply:
sorry, what am I missing after what..?

Reply:
</section>?

Reply:
The `<section anchor="MTI" title="Mandatory to Implement Hash Algorithm">` opens a section, which needs to be closed with `</section>` - which would follow the `</t>`.